frontend/components/Source.vue

```diff
@@ -85,6 +89,18 @@ export default {
                    </div>
                  </div>
 
+                 <div role="tabpanel" class="tab-pane" id="badge">
+                   <div class="main-nav">
+                     <h3>Image URL</h3>
+                     <pre>https://goodtables.io/badge/{{source.integration_name}}/{{source.name}}.svg</pre>
```

Comment:
I've done it on purpose to prevent leaking staging badges to github repos (e.g. Gabriela had a similar problem with hook activated for staging before hooks fix). WDYT?